lua/packer/plugin_types/git.lua

```diff
@@ -41,22 +41,32 @@ local function has_wildcard(tag)
   return string.match(tag, '*') ~= nil
 end
 
-local break_tag_pattern = [=[[bB][rR][eE][aA][kK]!?:]=]
+local break_tag_pattern = [=[[bB][rR][eE][aA][kK]:]=]
 local breaking_change_pattern = [=[[bB][rR][eE][aA][kK][iI][nN][gG][ _][cC][hH][aA][nN][gG][eE]]=]
+local type_exclam_pattern = [=[[a-zA-z]!:]=]
+local type_scope_exclam_pattern = [=[[a-zA-z]%([^)]+%)!:]=]
```

Comment:
```suggestion
local type_exclam_pattern = [=[[a-zA-Z]+!:]=]
local type_scope_exclam_pattern = [=[[a-zA-Z]+%([^)]+%)!:]=]
```

Reply:
In my prev comment I was suggesting to fix the A-Z.

Reply:
🤦
Fixed! (for real this time)